src/csharp/Microsoft.Spark/Sql/ArrowArrayHelpers.cs

```diff
@@ -41,59 +41,33 @@ internal static class ArrowArrayHelpers
 
         public static DataFrameColumn CreateEmptyColumn<T>()
         {
-            Type type = typeof(T);
-            if (type == typeof(BooleanDataFrameColumn))
-            {
-                return new BooleanDataFrameColumn("Empty");
-            }
-            else if (type == typeof(SByteDataFrameColumn))
-            {
-                return new SByteDataFrameColumn("Empty");
-            }
-            else if (type == typeof(ByteDataFrameColumn))
-            {
-                return new ByteDataFrameColumn("Empty");
-            }
-            else if (type == typeof(Int16DataFrameColumn))
-            {
-                return new Int16DataFrameColumn("Empty");
-            }
-            else if (type == typeof(UInt16DataFrameColumn))
-            {
-                return new UInt16DataFrameColumn("Empty");
-            }
-            else if (type == typeof(Int32DataFrameColumn))
-            {
-                return new Int32DataFrameColumn("Empty");
-            }
-            else if (type == typeof(UInt32DataFrameColumn))
-            {
-                return new UInt32DataFrameColumn("Empty");
-            }
-            else if (type == typeof(Int64DataFrameColumn))
-            {
-                return new Int64DataFrameColumn("Empty");
-            }
-            else if (type == typeof(UInt64DataFrameColumn))
-            {
-                return new UInt64DataFrameColumn("Empty");
-            }
-            else if (type == typeof(SingleDataFrameColumn))
-            {
-                return new SingleDataFrameColumn("Empty");
-            }
-            else if (type == typeof(DoubleDataFrameColumn))
-            {
-                return new DoubleDataFrameColumn("Empty");
-            }
-            else if (type == typeof(ArrowStringDataFrameColumn))
-            {
-                return new ArrowStringDataFrameColumn("Empty");
-            }
-            else
-            {
-                throw new NotSupportedException($"Unknown type: {typeof(T)}");
-            }
+            return typeof(T) switch
+            {
+                Type t when t == typeof(BooleanDataFrameColumn) =>
+                    new BooleanDataFrameColumn("Empty"),
+                Type t when t == typeof(SByteDataFrameColumn) =>
+                    new SByteDataFrameColumn("Empty"),
+                Type t when t == typeof(ByteDataFrameColumn) =>
+                    new ByteDataFrameColumn("Empty"),
+                Type t when t == typeof(Int16DataFrameColumn) =>
+                    new Int16DataFrameColumn("Empty"),
+                Type t when t == typeof(UInt16DataFrameColumn) =>
+                    new UInt16DataFrameColumn("Empty"),
+                Type t when t == typeof(Int32DataFrameColumn) =>
```

Comment:
missing uint32?

Reply:
Sorry.. I missed it.
I just add it!
 Thanks for letting me know.